static/app/views/settings/components/forms/fieldFromConfig.tsx

```diff
@@ -98,6 +99,8 @@ export default class FieldFromConfig extends Component<Props> {
         return <ProjectMapperField {...props} />;
       case 'sentry_project_selector':
         return <SentryProjectSelectorField {...props} />;
+      case 'select_async':
+        return <SelectAsyncField {...(props as any)} />;
```

Comment:
Can we type `props` here?

Reply:
@billyvg I tried to do this, but it's still giving me type errors Im not really sure how to type this correctly 😬

Reply:
update I think maybe now it does work? not sure I think my vscode is borking so maybe you could pull it down and see if Im just going loco